src/main/webapp/app/shared/dashboards/tutor-leaderboard/tutor-leaderboard.component.html

```diff
@@ -78,7 +78,12 @@
                     </ng-container>
 
                     <ng-container *ngIf="!isExerciseDashboard && course && course.id">
-                        <td>
+                        <td *ngIf="isExamMode && exam">
```

Comment:
```suggestion
                        <td *ngIf="isExamMode">
```
isExamMode is only true when there is an exam